core/pipeline.py

```diff
@@ -168,7 +169,8 @@ def output_dir(self):
 
     @property
     def report_template_dir(self):
-        return os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(sys.argv[0])))), "bfx", "report")
+        return os.path.join(os.path.dirname(os.path.dirname(os.path.abspath(inspect.getsourcefile(Pipeline)))), "bfx",
```

Comment:
Line change to create a path relative to core/pipeline.py, rather than the path of the pipeline. Using inspect will obtain the path of this file consistently.

Will no longer make assumptions on the location of the pipeline script.